apps/storefront/src/app/[locale]/(main)/search/page.tsx

```diff
@@ -85,11 +86,25 @@ export default async function Page({ searchParams }: PageProps) {
   );
   const { options } = searchService.getSortByOptions(searchContext);
 
+  const getHeader = () => {
+    if (query) {
+      return t("results-for", { query });
+    }
+
+    if (searchParams.category) {
+      return (
+        searchParams.category[0].toUpperCase() + searchParams.category.slice(1)
+      );
+    }
+
+    return null;
+  };
+
   return (
     <div className="w-full">
       <section className="mx-auto my-8 grid gap-8">
         <div className="flex items-center justify-between">
-          <h2 className="text-xl">{query && t("results-for", { query })}</h2>
+          <h1 className="text-2xl">{getHeader()}</h1>
```

Comment:
```suggestion
          getHeader() && <h1 className="text-2xl">{getHeader()}</h1>
```

+ same question as above with h1

Reply:
There is no conflict in lighthouse report. But it does not seem right - I changed h1 in navigation to h2.